### PR TITLE
Add onChange event to passwordInput

### LINKED ISSUE #4173: Add On change event in Password Input Component
### Is your proposal related to a problem?

Currently, there are no event handlers for the Password Input Component. We should have an event `on change` whenever the value in the component is changed.

### Describe the solution you'd like

The Password Input's configuration pane should have an "Events" section, just like other components.

Refer to this PR: https://github.com/ToolJet/ToolJet/pull/4058

### PR DESCRIPTION
**Pull request by MohammedBajuaifer**
closes #4173

### CODE REVIEW

**Comment by Navaneeth-pk** [comment]
@MohammedBajuaifer please add proper title for PRs.

**Comment by MohammedBajuaifer** [comment]
@Navaneeth-pk,  Updated.

**Comment by arpitnath** [comment]
@MohammedBajuaifer  Please can you fix the lint errors?

**Comment by MohammedBajuaifer** [comment]
Hey @arpitnath, updated the PR

**Review by arpitnath — DISMISSED** [review]

**Comment by Navaneeth-pk** [comment]
@MohammedBajuaifer  please resolve merge conflicts.

File: frontend/src/Editor/Components/PasswordInput.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export const PasswordInput = ({ height, validate, properties, styles, setExposedVariable, darkMode, component }) => {
+export const PasswordInput = ({ height, validate, properties, styles, setExposedVariable, darkMode, fireEvent }) => {
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Components/PasswordInput.jsx:3]:
```suggestion
export const PasswordInput = ({ height, validate, properties, styles, setExposedVariable, darkMode, component, fireEvent }) => {
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@MohammedBajuaifer You replaced the `component` prop which is breaking the component. Made a suggestion, can you please update the code?

**Review by kavinvenkatachalam — DISMISSED** [review]
LGTM

**Review by arpitnath — DISMISSED** [review]

**Comment by kavinvenkatachalam** [comment]
@MohammedBajuaifer Can you fix lint errors?

**Review by sherfin94 — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @MohammedBajuaifer 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**